src/main/kotlin/no/nav/syfo/varsel/AktivitetskravVarselPlanner.kt

```diff
@@ -27,70 +26,95 @@ class AktivitetskravVarselPlanner(
 
     private val log: Logger = LoggerFactory.getLogger("no.nav.syfo.varsel.AktivitetskravVarselPlanner")
     private val varselUtil: VarselUtil = VarselUtil(databaseAccess)
-    private val sykeforlopService: SykeforlopService = SykeforlopService()
 
     override suspend fun processOppfolgingstilfelle(aktorId: String, fnr: String) = coroutineScope {
         val oppfolgingstilfellePerson = syfosyketilfelleConsumer.getOppfolgingstilfelle(aktorId)
+        log.info("-$name-: oppfolgingstilfellePerson for -$aktorId- aktor id er -$oppfolgingstilfellePerson-")
 
-        if(oppfolgingstilfellePerson == null) {
-            log.info("[$name]: Fant ikke oppfolgingstilfelle for denne brukeren. Planlegger ikke nytt varsel")
+        if (oppfolgingstilfellePerson == null) {
+            log.info("-$name-: Fant ikke oppfolgingstilfelle for -$aktorId- aktor id. Planlegger ikke nytt varsel")
             return@coroutineScope
         }
 
         val gyldigeSykmeldingTilfelledager = oppfolgingstilfellePerson.tidslinje.stream()
```

Comment:
I Kotlin trenger man ikke konvertere til stream og tilbake list igjen som i Java.
Kan skrive om til:
`        val gyldigeSykmeldingTilfelledager = oppfolgingstilfellePerson.tidslinje
            .filter { isGyldigSykmeldingTilfelle(it) }`